faqs.rst

```diff
@@ -50,24 +50,38 @@ or `sched <https://github.com/flux-framework/flux-sched/issues>`_.
 
 .. _not_managing_all_resources:
 
-Why is Flux not discovering and managing all of the resources on the system/node?
-=================================================================================
+Why is Flux ignoring my Nvidia GPUs?
+====================================
 
-This can be due to various bind flags that need to be passed parallel launcher
-that started Flux. For example at LLNL you must pass ``--mpibind=off`` to
-``srun`` and ``--bind=none`` to ``jsrun``.
+When Flux is launched via a foreign resource manager like SLURM or LSF,
+it must discover available resources from scratch using
+`hwloc <https://www.open-mpi.org/projects/hwloc/>`_.  To print a resource
+summary, run:
 
-Also on all systems, Flux relies on hwloc to auto-detect the on-node resources
-available for scheduling.  The hwloc that Flux is linked against must be
-configured with ``--enable-cuda`` for Flux to be able to detect Nvidia GPUs.
+.. code-block:: sh
+
+  $ flux resource list
+       STATE NNODES   NCORES    NGPUS NODELIST
+        free     16       96       16 test[0-15]
+   allocated      0        0        0
+        down      0        0        0
```

Comment:
Suggestion:  We do support `flux resource info`, and its output is more compact.